lib/ui/nav_pages/you_page/you_page_viewmodel.dart

```diff
@@ -70,6 +80,33 @@ class YouPageViewModel extends BaseViewModel {
     await _navigationService.navigateTo(Routes.preferenceView);
   }
 
+  void navigateToSignIn() =>
+      navigationService.pushNamedAndRemoveUntil(Routes.loginView);
+
+  String? get token => _storage.getString(StorageKeys.currentSessionToken);
+
+  Future<void> signOutAccount() async {
+    bool connected = await _connectivityService.checkConnection();
+    const endpoint = "/auth/logout";
+    if (!connected) {
+      _snackBar.showCustomSnackBar(
+          message: "No internet connection, connect and try again.",
+          variant: SnackbarType.failure,
+          duration: const Duration(milliseconds: 1500));
+      return;
+    }
+
+    final response = await _apiService.post(endpoint, body: {}, token: token);
+
+    if (response?.statusCode == 200) {
```

Comment:
This should rather just sign out from the current organization and clear all route history while navigating back to the organisation view page

Reply:
Changes Implemented